recipes/roary/build.sh

```diff
@@ -20,4 +20,6 @@ else
     exit 1
 fi
 
+wget https://raw.githubusercontent.com/sanger-pathogens/Roary/master/bin/create_pan_genome_plots.R -P $PREFIX/bin/
+
 chmod u+rwx $PREFIX/bin/*
```

Comment:
I'm pretty sure this won't work. You can't change permissions on dependencies without throwing a file permissions error.